WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -878,6 +879,10 @@ private extension ProductsViewController {
             }
             .store(in: &subscriptions)
 
+        updateBlazeBannerVisibility()
```

Comment:
nit: I wasn't completely sure about the reason why this was separated into a new function. How about moving this call `updateBlazeBannerVisibility` out of this function and calling it whenever necessary, and then renaming `checkBlazeBannerVisibility` to like `observeBlazeBannerVisibility` which then can be just called once?

Reply:
Good idea - updated in d6f6e7180715839c24c3e88e63b5fd57c60ece55.